### ISSUE TITLE
Output of strace is not parsed correctly when attaching to a process

### ISSUE THREAD

**Issue by daschubert** [SiegelDaniel/MQTT-IDS#8]
When starting to trace a process, the first message on the "TRACED" topic is similar to this:
```
<trace_data><pinfo>2769</pinfo><trace>strace: Process 2769 attached</trace></trace_data>
```
This results in the following message on the "REFINED" topic:
```
<Refined><Timestamp>strace:</Timestamp><Syscall>Process 2769 attached</Syscall><PInfo>2769</PInfo></Refined>
```

This is obviously not consistent with the intended format.

**Comment by SiegelDaniel** [comment]
Fixed in 88b14f9e6f241209d806560d69f7e14ac6742483, ready for testing.

**Comment by daschubert** [comment]
The fix is fine for the described problem. However, I guess we should generally discard messages that do not adhere to the specified format, e.g. I get the following message on REFINED when dataching:
```
<Refined><Timestamp></Timestamp><Syscall>&lt;detached ...&gt;</Syscall><PInfo>2008</PInfo></Refined>
```
I guess we need a shema and let the parser validate. However, we can do that in another issue and close this one once the above is fixed.

**Comment by SiegelDaniel** [comment]
Added a quick and dirty fix for the above in 50b64c941c5f97db4553dad8dc0495bd18b160c3
Will reference the above in a following issue
